MobileApplication/MobileApplication/Views/AboutPage.xaml.cs

```diff
@@ -21,6 +21,10 @@ public partial class AboutPage : ContentPage
     {
         private NewCategoryPage _newCategory;
         public CategoryPage categoryPageNew;
+        private float b;
```

Comment:
What is b? The names should be more informative.

Reply:
Okay, I will change it